tools/packaging/debian/control

```diff
@@ -0,0 +1,17 @@
+Source: dragonfly
+Maintainer: DragonflyDB authors  <ragonfly@dragonflydb.io>
+Standards-Version:  4.2.1
+Priority: optional
+Section: database
+Vcs-Git: https://github.com/dragonflydb/dragonfly
+
+Package: dragonfly
+Architecture: amd64 arm64
+Depends: libc6, openssl, adduser, systemctl
+Homepage: https://dragonflydb.io
+Description: It is probably the fastest memory store in the universe.
+  Persistent key-value database with network interface.
```

Comment:
lets replace it  with:
`A fast in-memory store that is fully compatible with Redis™* and Memcached`

Reply:
sure